website/content/docs/connect/config-entries/exported-services.mdx

```diff
@@ -379,6 +379,12 @@ spec:
 
 The following example configures Consul to export the `payments` and `refunds` services from the `billing` namespace of the `finance` admin partition to the `web-shop` partition.
 
+When exporting to datacenter-local admin partitions rather than a cluster peer,
```

Comment:
I agree that "datacenter-local" isn't a term that's defined on this page (or used widely elsewhere in our docs).

My concern is that, since the introduction of admin partitions, I don't think "local" has an unambiguous meaning anymore. Does "local" mean "the current partition"? Or "the current datacenter"? (Or something else?)

This was my attempt at disambiguating: datacenter-local = in the same datacenter.

So this could instead be:

```suggestion
When exporting to other admin partitions in the same datacenter,
```

Reply:
Tangential: We'll likely have the same conversation about the "datacenter-local" term in the DNS docs improvements we've been working on.